clients/nasaClient.ts

```diff
@@ -0,0 +1,18 @@
+import dotenv from "dotenv"
```

Comment:
Normally I wouldn't expect you to need the `dotenv` package and it should just be possible to directly call with `process.env`. Let me know if that isn't the case tho

Reply:
Without this line I get a red underline and this message: Cannot find name 'dotenv'.ts(2304)

Reply:
Without this, axios returns a 403 response and in the error message shows the url used as having `api_key=undefined`
I followed steps from here to solve this: https://www.geeksforgeeks.org/how-to-access-expressjs-env-variables-in-an-api-endpoint/

Reply:
ahh you're right, looks like express doesn't automatically import .env files. I think other frameworks I've worked with automatically do this (or they're somewhere in a setup). This is fine then :D